styleguide/source/_patterns/02-organisms/03-sections/07-simple-step-form/simple-step-form.twig

```diff
@@ -0,0 +1,32 @@
+{#
+ # @file simple-step-form.twig
+ # Copyright 2017 Palantir.net, Inc.
+ #}
+
+<form class="simple-step-form" method="get" action="" novalidate="novalidate">
+  <fieldset>
+    <p class="h2">To reach us by email, please fill out the following:</p>
+    {% include 'molecules-input-text' %}
+    {% include 'molecules-input-telephone' %}
```

Comment:
I'm not sure if this is required functionality or not, but when I enter a phone number that is letters, it validates it as "okay". 

<img width="751" alt="screen shot 2017-02-27 at 4 23 53 pm" src="https://cloud.githubusercontent.com/assets/4439057/23382850/6a3b3d10-fd09-11e6-86be-0a20a17f650b.png">